### ISSUE TITLE
make_figures.smk produces a MissingInputException 

### ISSUE THREAD

**Issue by gtsambos** [quinlan-lab/proj-mutator-mapping#2]
Thanks so much for this very comprehensive reproducible pipeline, Tom ✨  I just thought I'd let you know that I get the following error when I do a dry run of `make_figures.smk`:

```
> snakemake -j1 -s scripts/make_figures.smk --dry-run
Building DAG of jobs...
MissingInputException in rule plot_wild_afs in file /Users/gt/projects/proj-mutator-mapping/scripts/rules/process_wild_data.smk, line 1:
Missing input files for rule plot_wild_afs:
    output: figs/wild_afs.eps
    wildcards: ext=eps
    affected files:
        data/vcf/wild.vcf.gz
``` 

(btw, I don't need this file to do my work, so no need to fix on my behalf! I just thought I'd make sure you're aware of it)

**Comment by tomsasani** [comment]
Thanks @gtsambos for opening up an issue for this! I never actually pushed `data/vcf/wild.vcf.gz` up to GitHub since it's fairly large (~150 Mb), and forgot to update the `make_figures.smk` pipeline accordingly. I'll probably create a smaller subsetted VCF and push that to Git.

**Comment by tomsasani** [comment]
Should be addressed by 0fc70dd and 0aec88b.
